experimental/devices/mfrc522/commands/low_level.go

```diff
@@ -35,14 +35,24 @@ type AuthStatus byte
 
 // NewLowLevelSPI creates and initializes the RFID card reader attached to SPI.
 //
-// 	spiPort - the SPI device to use.
-// 	resetPin - reset GPIO pin.
-// 	irqPin - irq GPIO pin.
+//  spiPort - the SPI device to use.
+//  resetPin - reset GPIO pin.
+//  irqPin - irq GPIO pin.
 func NewLowLevelSPI(spiPort spi.Port, resetPin gpio.PinOut, irqPin gpio.PinIn) (*LowLevel, error) {
+	return NewLowLevelSPIFrequency(spiPort, resetPin, irqPin, 10*physic.MegaHertz)
+}
+
+// NewLowLevelSPIFrequency creates and initializes the RFID card reader attached to SPI.
+//
+//  spiPort - the SPI device to use.
+//  resetPin - reset GPIO pin.
+//  irqPin - irq GPIO pin.
+//  frequency - the SPI frequency
+func NewLowLevelSPIFrequency(spiPort spi.Port, resetPin gpio.PinOut, irqPin gpio.PinIn, frequency physic.Frequency) (*LowLevel, error) {
```

Comment:
Technically, if the user wants a lower frequency that the maximum supported by the device, they should use instead spi.PortCloser.LimitSpeed(). I agree it's not obvious (thus I'm open to improved examples) but it's better than adding similar code to every single driver.

https://periph.io/x/periph/conn/spi#PortCloser

Reply:
```go
type LowLevel struct {
	resetPin    gpio.PinOut
	irqPin      gpio.PinIn
	spiDev      spi.Conn
	antennaGain int
	stop        chan struct{}
}
```
`spiDev` is unexported, so it's impossible to do that. Maybe we could export it and remove the `NewSPIFrequency` method?

Reply:
Here's an example for a client: https://github.com/google/periph/blob/master/cmd/apa102/main.go#L154

Reply:
Thanks, I made it work using `LimitSpeed`. I didn't know about that method. I will remove the two new functions.